scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -304,7 +304,7 @@ def run_build(self, args=[]):
                     log.write(log_msg)
 
             if log_msg:
-                overflow_found = re.findall("region `(FLASH|ROM|RAM|ICCM|DCCM|SRAM|dram\\d_\\d_seg)' overflowed by", log_msg)
+                overflow_found = re.findall("region [`'](FLASH|ROM|RAM|ICCM|DCCM|SRAM|dram\\d_\\d_seg)':? overflowed by", log_msg)
```

Comment:
is it possible to add a test for this enhancement?

Reply:
I think there might be an existing one, but I believe there are some complications in testing with lld at the moment--please see my response below!